tests/grpc_service_storage.rs

```diff
@@ -66,7 +66,7 @@ async fn grpc_service_storage() -> Result<(), Box<dyn std::error::Error>> {
         "127.0.0.1".to_string(),
         12300,
         "amqp://guest:guest@localhost:5672".to_string(),
-        "http://guest:guest@localhost:15672/api".to_string(),
+        Some("http://guest:guest@localhost:15672/api".to_string()),
```

Comment:
just thinking: now that we have two MQ backends, maybe we should start to consider applying a testing matrix?